.travis.yml

```diff
@@ -6,6 +6,8 @@ python:
   - 3.6
   - 3.7
   - 3.8
+  - 3.9
+  - 3.10
```

Comment:
Was doing development in 3.10 -- update the documentation to mention that it supports up to 3.10